### PR TITLE
story/WEB-61

### PR DESCRIPTION
**Pull request by ajay3568yadav**
removed researchers:
Sola
David

Remove position on opportunities page (comment out)

moved Vishnu to alumni

### CODE REVIEW

File: _includes/map.html [hunk 1]
```diff
@@ -56,27 +56,29 @@ <h2 class="separate">Where we are from</h2>
...
-  ["David Costello", 39.063900, -108.550600],
   ["Namita Shah", 33.297139, -111.935985]
   ];
 
   let alumni = [
   ["Sudheera Daggumalli", 15.9129, 79.7400],
+  ["David Costello", 39.063900, -108.550600],
+  ["Vishnu Saniko", 15.404126, 79.508372],
+  ["David Costello", 39.063900, -108.550600],
+  ["Vishnu Saniko", 15.404126, 79.508372],
```
Comment (jdamerow) [inline _includes/map.html:71]: duplicate lines

File: opportunities/index.html [hunk 2]
```diff
@@ -21,16 +21,16 @@
...
+	<!--<p>We are looking for several highly-motivated graduate and upper-level undergraduate students to join our team. For posting information and instructions how to apply, please consult the postings on the ASU student employment website. Please do not send individual emails to us. We can only consider applications submitted through the ASU student employment site. Simply follow the instructions below: </p-->
+	<!--<ol style="list-style-type: decimal; padding-left: 15px;">
 
 		<li>Visit <a href="https://students.asu.edu/employment/search" target="_blank">https://students.asu.edu/employment/search</a>.</li>
 		<li>Click on "Search On-Campus Jobs".</li>
 		<li>Click on "Search for Openings".</li>
 		<li>Search for requisition ID <strong>97498BR</strong>.</li>
 		<li>Follow the instructions listed.</li>
-	</ol>
+	</ol-->
```
Comment (jdamerow) [inline opportunities/index.html:33]: should be a proper comment ending, `</ol>-->

**Review by jdamerow — CHANGES_REQUESTED** [review]

File: opportunities/index.html (hunk 2 above)
Comment (ajay3568yadav) [reply opportunities/index.html:33]: fixed

File: _includes/map.html (hunk 1 above)
Comment (ajay3568yadav) [reply _includes/map.html:71]: fixed

File: _includes/alumni.html [hunk 3]
```diff
@@ -5,7 +5,7 @@
       <div class="mod modSectionHeader ">
         <div class="special-title centered-text separateline">
           <a name="alumni"></a>
-          <h2 class="separate" style="color: ;">Alumni</h2>
+          <h2 class="separate" style="color: black;">Alumni</h2>
```
Comment (jdamerow) [inline _includes/alumni.html:8]: i don't think the headers are black. Just remove the style attribute here.

**Review by jdamerow — CHANGES_REQUESTED** [review]

File: _includes/alumni.html (hunk 3 above)
Comment (ajay3568yadav) [reply _includes/alumni.html:8]: fixed

File: _includes/alumni.html [hunk 4]
```diff
@@ -5,7 +5,7 @@
       <div class="mod modSectionHeader ">
         <div class="special-title centered-text separateline">
           <a name="alumni"></a>
-          <h2 class="separate" style="color: ;">Alumni</h2>
+          <h2 class="separate" style="color:;">Alumni</h2>
```
Comment (jdamerow) [inline _includes/alumni.html:8]: just remove the whole style attribute

**Review by jdamerow — CHANGES_REQUESTED** [review]

File: _includes/alumni.html (hunk 4 above)
Comment (ajay3568yadav) [reply _includes/alumni.html:8]: removed style attribute

**Pull request merged**